ext/bin/run-load-test

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+
+from sys import exit, stderr, stdout
+from datetime import datetime
+import os, sys, getopt, subprocess, shutil
+
+def usage(stream):
+    usage = """
+Usage: run-load-test --host http://myhost:port [options]
+
+Options:
+  -h [--host]       Host to which the requests are made
+  -u [--users]      Number of users to simulate. Default 1
+  -s [--spawn_rate] The rate per second in which users are spawned. Default 1
+  -t [--timelimit]  How long the swarm should take. Default 1m (Examples: 30s, 1h30m)
```

Comment:
This argument should be named `--run-time` to be consistent with the argument from locust

Reply:
Updated, now all arguments are passed directly to locust.